.babelrc

```diff
@@ -9,5 +9,5 @@
     }],
     "react"
   ],
-  "plugins": ["transform-object-rest-spread"]
+  "plugins": ["transform-object-rest-spread", "transform-class-properties"]
```

Comment:
I :heart: this (cf. https://babeljs.io/docs/plugins/transform-class-properties/)